VaticanBAV/et/BAVet119.xml

```diff
@@ -0,0 +1,552 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+   schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BAVet119" type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+          <titleStmt>
+              <title xml:lang="gez" xml:id="t1">ዐውደ፡ ነገሥት፡</title>
+              <title xml:lang="gez" corresp="#t1" type="normalized">ʿAwda nagaśt</title>
+              <title xml:lang="en" corresp="#t1">Cycle of the kings and other treatises and tables on divination and astrology</title>
+              <editor key="MV"/>
+              <editor key="AB" role="generalEditor"/>
+              <funder>Akademie der Wissenschaften in Hamburg</funder>
+          </titleStmt>
+          <publicationStmt>
+              <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+              <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+              <pubPlace>Hamburg</pubPlace>
+              <availability>
+                  <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                      licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+              </availability>
+          </publicationStmt>
+          <sourceDesc>
+              <msDesc xml:id="ms">
+                  <msIdentifier>
+                      <repository ref="INS0003BAV"/>
+                      <collection>Aethiopici</collection>
+                      <idno>Aeth.119</idno>
+                  </msIdentifier>
+                 
+                  <msContents>
+                      <summary/>
+                      <msItem xml:id="ms_i1">
+                          <locus from="3r" to="36v"/>
+                          <title type="complete" ref="LIT1171Awdana">ʿAwda nagaśt</title>
+                          <msItem xml:id="ms_i1.1">
+                              <locus target="#3r"/>
+                              <title type="complete">List of names and letters with which portions of the circle are annotated.</title>
+                          </msItem>
+                          <msItem xml:id="ms_i1.2">
+                              <locus target="#3v"/>
+                              <title>Figure consisting of a cross with four human faces at the ends 
```

Comment:
This is a bit strange for a title.

Is this item a text? Then have a short title and a description to it

If this is a regular component of `LIT1171Awdana `then probably a div should be created there to describe this all and here you then only have a ref with `LIT1171Awdana#...`

Reply:
Probably <title>Magic picture</title> and <desc>Figure consisting of a cross with four human faces at the ends of the four arms and an eye in the middle of each arm. Four circles, divided into four parts, fill the space between the arms. Future events are indicated with this figure.</desc>

Reply:
> This is a bit strange for a title.
> 
> Is this item a text? Then have a short title and a description to it
> 
> If this is a regular component of `LIT1171Awdana `then probably a div should be created there to describe this all and here you then only have a ref with `LIT1171Awdana#...`

Other LIT1171Awdana manuscript do not have this introductory figure, which is why I didn't create a div, deeming the figure an occasional addition. 

I have moved this figure into decoration because, although the cataloguers state that the purpose of the figure is divinatory and not decorative, there is nothing to suggest that there is any written text here.